Source/WSTagsField.swift

```diff
@@ -326,8 +326,19 @@ open class WSTagsField: UIScrollView {
         addTag(WSTag(tag))
     }
 
+    open var validator: ((WSTag, [WSTag]) -> Bool)?
+
+    open var ignoreCaseWhenAdding: Bool = false
```

Comment:
@Pe-te For the framework to be, at the same time, more straightforward and flexible, what to you think to just keep the `validator`?

Reply:
Oh BTW, I think the `WSTagsField` has an event for that.

Reply:
Those [`onShouldAcceptTag`](https://github.com/whitesmith/WSTagsField/blob/14338e327015da58166adb032e772efbc936d139/Source/WSTagsField.swift#L190) helps for what do you need?

Reply:
Hmm, maybe not. Please remove `ignoreCaseWhenAdding` and rename `validator` to `onValidateTag` (put it in the Events section of the code). Thanks a lot!